CHECKLIST.md

```diff
@@ -3,51 +3,63 @@
 The following is a list of common things that should be checked when
 submitting a pull request.
 
-- [ ] Javadoc classes and methods
 
-- [ ] All changes are unit tested
+## Common
 
 - [ ] Include screenshots and animated GIFs in your pull request
     whenever possible.
 
 - [ ] Format your code using the default NetBeans formatter settings.
     For specifics see the [style guide](STYLE_GUIDELINES.md).
 
-- [ ] Ensure that new ***leaks*** or ***code smells*** are not
-    introduced by parsing you code using `sonar-scanner` and checking
-    via [Sonar Qube](https://sonarcloud.io)
-
 - [ ] Document code based on the [style guide](STYLE_GUIDELINES.md)
 
-- [ ] Avoid OS dependant code (e.g. Windows or Linux)
+- [ ] Javadoc classes and methods
 
-- [ ] Utility methods should end with the word Utilities (i.e not Util
-    or Utility)
+- [ ] Unit testing changes in your PR
+
+    - Unit testing changes you have made is required and should be
+    over 80% code coverage as a minimum.
+
+    - If any classes you have changed do not already have a unit tests
+    then it is **NOT** apon you to test the entire class, rather the
+    delta changes only.
+
+    - Refer to the `Core Table View` as a baseline of how tests should
+    be written.
 
-- [ ] Update the [change log](CHANGELOG.md) as a means to notify
-    developers or important improvements or changes
+    - Use `Mockito`, `FxToolkit` and `FxRobot` as required.
+
+- [ ] Update the [change log](CHANGELOG.md) to inform developers of
+    important improvements or changes
 
     -   You must start with Added, Fixed, Improved, Moved, Removed,
         Renamed or Updated and sort them alphabetically
 
-- [ ] Update whatsnew.txt if you want to inform users about your change
+- [ ] Update `whatsnew.txt` if you want to inform users of your change
```

Comment:
A link to whatsnew.txt would be nice, similar to changelog above. (for consistency)

Reply:
No worries @aldebaran30701 that has been updated. Let me know if you can think of anything else.